scss/base/buttons.scss

```diff
@@ -250,3 +250,27 @@ a.text-link.small {
   font-size: $font-size-sm;
   padding: 18px 40px;
 }
+
+.btn-default-dark,
+.btn-default-dark.active {
```

Comment:
Hm, let's talk about this - maybe we can avoid making extra button extensions to keep our style guide simple.